dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentParamsMapper.java

```diff
@@ -158,20 +160,21 @@ private void validateTrackedEntity( String id, TrackedEntity trackedEntity )
         }
     }
 
-    private Set<OrganisationUnit> validateOrgUnits( User user, Set<String> orgUnitIds, Program program )
+    private Set<OrganisationUnit> validateOrgUnits( User user, Set<UID> orgUnitIds, Program program )
         throws BadRequestException,
         ForbiddenException
     {
         Set<OrganisationUnit> orgUnits = new HashSet<>();
         if ( orgUnitIds != null )
         {
-            for ( String orgUnitId : orgUnitIds )
+            for ( UID orgUnitUid : orgUnitIds )
```

Comment:
I guess the variable name can now drop the `Uid` suffix.

Reply:
I can't as its taken up by the `orgUnit` variable. I also don't want to spend much time in here as we are sweeping through it again https://dhis2.atlassian.net/browse/TECH-1534
We will move this logic into the service soon.